app/uk/gov/hmrc/exports/models/declaration/Locations.scala

```diff
@@ -54,6 +55,15 @@ object WarehouseIdentification {
 case class OfficeOfExit(officeId: String, presentationOfficeId: Option[String], circumstancesCode: Option[String])
 object OfficeOfExit {
   implicit val format: OFormat[OfficeOfExit] = Json.format[OfficeOfExit]
+  def from(officeOfExitSupplementary: OfficeOfExitSupplementary): OfficeOfExit =
```

Comment:
Why are these required? If the front end sends us an `OfficeOfExit` and the back end just needs an `OfficeOfExit` surely we can get rid of this?

Reply:
Fixed this for you @PauloPortugal